project/components/resource_request/schemas.py

```diff
@@ -25,7 +27,9 @@ class ResourceRequestSchema(BaseSchema):
     """General resource request schema."""
 
     project_id: UUID
-    requested_by_user_id: str = ''
+    user_id: str = ''
+    email: EmailStr = ''
+    username: str = ''
     requested_for: str = ''
```

Comment:
Please remove all default values, since all of them must be provided upon creation.
```suggestion
    user_id: str
    email: EmailStr
    username: str
    requested_for: str
```

Reply:
I just did like that because it was setting this default before:
`requested_by_user_id: str = ''`